app/views/about/index.volt

```diff
@@ -1,14 +1,8 @@
-<div align="center">
+<h2 class="mt-5">About this Demo</h2>
 
-	<div align="left">
-		<h2>About this Demo</h2>
-	</div>
-
-	<div align="left">
-		This is a sample application for the <a href="http://phalconphp.com">Phalcon Framework</a>. We expect to implement as many features
-		as possible to show how the framework works and its potential. Please write us if you have any
-		feedback or comments. Feel free to clone the code of this application <a href="https://github.com/phalcon/vokuro">here</a>.
-		The purpose of this application is merely academic. Thanks!.
-	</div>
-
-</div>
\ No newline at end of file
+<div>
+    This is a sample application for the <a href="http://phalconphp.com">Phalcon Framework</a>. We expect to implement as many features
```

Comment:
I think we can start using phalcon.io as our main domain. CC @niden

Reply:
Yes please replace phalconphp.com with phalcon.io.

Reply:
77f064b